lib/opbeat/rack.rb

```diff
@@ -28,16 +28,14 @@ def call(env)
       rescue Error => e
         raise # Don't capture Opbeat errors
       rescue Exception => e
-        evt = Event.from_rack_exception(e, env)
-        Opbeat.send(evt)
+        Opbeat.capture_rack_exception(e, env)
```

Comment:
Becouse it is sent in the `capture_rack_exception` method. Also I think that original method didn't support `async` and also didn't support ignoring user defined errors. (note: it was also not checking if `evt` is nil)